tests/benchmarks/power_consumption/uart_polling/src/driver_test.c

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr/drivers/uart.h>
+#define UART_DEVICE_NODE DT_CHOSEN(zephyr_console)
+static const struct device *const uart_dev = DEVICE_DT_GET(UART_DEVICE_NODE);
```

Comment:
This can be single line ie DEVICE_DT_GET(DT_CHOSEN(zephyr_console))
without #define above

Reply:
ok, lets make it single line